src/Illuminate/Validation/Rules/File.php

```diff
@@ -13,10 +13,20 @@
 use Illuminate\Support\Traits\Macroable;
 use InvalidArgumentException;
 
-class File implements Rule, DataAwareRule, ValidatorAwareRule
+class File implements DataAwareRule, Rule, ValidatorAwareRule
 {
     use Conditionable, Macroable;
 
+    /**
+     * Binary units flag used for size validation.
+     */
+    protected const BINARY = 'binary';
+
+    /**
+     * International units flag used for size validation.
+     */
+    protected const INTERNATIONAL = 'international';
```

Comment:
Wouldn't it be better to make this an enum?

Reply:
I'm asking, among other things, because this ties in with https://github.com/laravel/framework/pull/56223#discussion_r2195683800, meaning that it wouldn't necessarily only be used in this one place